.circleci/config.yml

```diff
@@ -175,6 +176,30 @@ commands:
             echo "Canceling workflow as previous step resulted in failure."
             echo "To execute all checks locally, please run yarn ci-tests"
             curl -X POST --header "Content-Type: application/json" "https://circleci.com/api/v2/workflow/${CIRCLE_WORKFLOW_ID}/cancel?circle-token=${WORKFLOW_CANCELER}"
+  list-recent-merges:
+    description: 'Lists the merge commits from the past 24 hours'
+    steps:
+      - run:
+          name: Listing merge commits from the past 24 hours
+          when: on_fail
+          command: |
+            git log --merges --oneline --since="24 hours ago" --pretty=format:"%h %<(12)%ar | %s [%an]" | grep "Merge pull request"
+  report-workflow-on-failure:
+    description: 'Reports failures to discord'
+    parameters:
+      template:
+        description: |
+          Which template to report in discord
+        type: string
+    steps:
+      - run:
+          when: on_fail
+          command: git fetch --unshallow
+      - list-recent-merges
+      - discord/status:
+          only_for_branches: chore/failure-reports
```

Comment:
This is nice, I think if we do this only for the `next` and `alpha` branch, we have exactly what we want.

I first thought about only notifying on the `merged` and `daily` workflow, but that would also notify when you run the `daily` workflow in your PR.

So setting this to `next would be better I guess.

Reply:
I had a similar thought and I think it makes sense too, I'll leave both next and alpha branches there, possibly updating to main in the future too 🤔